src/config.rs

```diff
@@ -49,5 +49,22 @@ pub fn get_backups_flder() -> PathBuf {
 
 #[cfg(target_family = "unix")]
 pub fn get_boilr_links_path() -> PathBuf {
-    get_config_folder().join("links").to_path_buf()
+    // Remove redundant_clone. Refer: https://rust-lang.github.io/rust-clippy/master/index.html#redundant_clone
```

Comment:
No reason to have a comment that something was removed. 
Its a good change, but no need to add a comment.

```suggestion
```